templates/launch.html

```diff
@@ -1,215 +1,405 @@
-                     alt="Reload symbol">
-            </span>
-        </div>
-        <h3>Runner Data</h3>
-        <div class="field-container">
-            <label for="exp">Token Expiry (seconds)</label>
-            <input id="exp" type="text" value="1800" class="qa-token-expiry">
-        </div>
-        <div class="field-container">
-            <label for="language_code">Language</label>
-            <select id="language_code" name="language_code" class="qa-language-code">
-                <option value="en">English (en)</option>
-                <option value="cy">Cymraeg (cy)</option>
-                <option value="ga">Gaeilge (ga)</option>
-                <option value="eo">Ulstér Scotch (eo)</option>
-                <option value="">&lt;not set&gt;</option>
-            </select>
-        </div>
-        <div class="field-container">
-            <label for="roles">Roles</label>
-            <select id="roles" name="roles" multiple="multiple" class="qa-roles">
-                <option value="flusher">flusher</option>
-                <option value="dumper" selected>dumper</option>
-            </select>
-        </div>
-        <div class="field-container">
-            <label for="account_service_url">Account Service URL</label>
-            <input id="account_service_url"
-                   name="account_service_url"
-                   type="text"
-                   value="{{.AccountServiceURL}}"
-                   class="qa-account_service_url">
-        </div>
-        <div class="field-container">
-            <label for="account_service_log_out_url">Account Service Log Out URL</label>
-            <input id="account_service_log_out_url"
-                   type="text"
-                   value="{{.AccountServiceLogOutURL}}"
-                   class="qa-account_service_log_out_url">
-        </div>
-        <div class="field-container">
-            <input type="submit"
-                   value="Open Survey"
-                   class="qa-btn-submit-dev btn btn--hidden"
-                   id="submit-btn"
-                   onclick="saveResponseId()" />
-            <input type="submit"
-                   name="action_flush"
-                   value="Flush Survey Data"
-                   class="qa-btn-submit-dev btn btn--hidden"
-                   id="flush-btn" />
-            <input type="button"
-                   value="Clear Local Storage"
-                   class="qa-btn-submit-dev btn"
-                   id="local-storage-btn"
-                   onclick="clearLocalStorage()" />
+        <div class="ons-grid">
+            <div class="ons-grid__col ons-col-6@m ">
```

Comment:
```suggestion
            <div class="ons-grid__col ons-col-6@l ">
```
I think we talked about this but I think it could be nice if the panels went under each other at a larger breakpoint. Line 117 will also need the same change. Try this and see what you think. I think its an improvement.

Reply:
I made that change I used `@xl` instead as it makes the buttons appear less squished, it looks better than before